PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetExtractor.cxx

```diff
@@ -792,6 +792,40 @@ Bool_t AliAnalysisTaskJetExtractor::Run()
       FillHistogram("hJEWEL_ImpactParameter", fImpactParameter);
     }
   }
+  // Perform the matching before the main jet loop
+  AliJetContainer * jetsHybrid = GetJetContainer(0);
+  AliJetContainer * jetsDetLevel = GetJetContainer(1);
+  AliJetContainer * jetsPartLevel = GetJetContainer(2);
+  // Validation                                                                                                                                      
+  if (!jetsHybrid) {
+    AliErrorStream() << "Could not retrieve hybrid jet collection.\n";
+    return kFALSE;
+  }
+  if (!jetsDetLevel) {
+    AliErrorStream() << "Could not retrieve det level jet collection.\n";
+    return kFALSE;
+  }
+  if (!jetsPartLevel) {
+    AliErrorStream() << "Could not retrieve part level jet collection.\n";
+    return kFALSE;
+  }
+  // Now, begin the actual matching.                                                                                                                 
+  // Hybrid <-> det first                                                                                                                            
+  AliDebugStream(2) << "Matching hybrid to detector level jets.\n";
+  // First, we reset the tagging                                                                                                                     
+  ResetMatching(*jetsHybrid);
+  ResetMatching(*jetsDetLevel);
+  // Next, we perform the matching                                                                                                                   
+  PerformGeometricalJetMatching(*jetsHybrid, *jetsDetLevel, (jetsHybrid->GetJetRadius()*0.75));
```

Comment:
For R = 0.2 jets, a max matching distance of 0.15 seems tighter than I would have expected. For example, I used 0.3 for R = 0.2 jets. I'm not sure that it will have a big impact, but perhaps something to think about. Perhaps worth making it configurable?

Reply:
Updated so this is a parameter.